src/main.cpp

```diff
@@ -780,6 +780,10 @@ Use Ctrl+C to stop.)");
   -m, --muxer               Set the output format to a specific muxer instead of detecting it
                             from the filename.
 
+  -s, --show-mouse          Shows the mouse in the output file. This is the default behavior.
+
+  -e, --hide-mouse          Hides the mouse in the output file.
```

Comment:
I find `-e` a bit unintuitive, maybe we can make this a long option only?

Reply:
The e is for --hid(E)-mouse.